dashboard/app/views/pd/workshop_mailer/_teacher_enrollment_details.html.haml

```diff
@@ -61,3 +61,9 @@
   P.S. Connect with other computer science teachers on Twitter
   = link_to '@TeachCode', 'https://twitter.com/teachcode', target: "_blank"
   for inspiration and great tips on bringing computer science to every school. Speaking of social media, if you don't want your photo taken or posted at the workshop, make sure the facilitator knows before the workshop begins.
+
+  P.S. Connect with other computer science teachers on Twitter
```

Comment:
should the previous paragraph be deleted?

Reply:
Great catch!